Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -113,13 +116,15 @@ def __init__(
                 This is used when collecting a pack containing a content item, when their marketplace values differ.
         :param only_to_install: whether to collect the pack only to install it without upload to the bucket.
         :param only_to_upload: whether to collect the pack only to upload it to the bucket without install.
+        :param pack_to_reinstall: pack name to collect for reinstall test
         """
         self.tests: set[str] = set()
         self.modeling_rules_to_test: set[str | Path] = set()
         self.packs_to_install: set[str] = set()
         self.packs_to_upload: set[str] = set()
         self.version_range = None if version_range and version_range.is_default else version_range
         self.machines: tuple[Machine, ...] | None = None
+        self.packs_to_reinstall_test: set[str] = set()
```

Comment:
```suggestion
        self.packs_to_reinstall: set[str] = set()
```
because it sounds like to want to reinstall a test